addons/l10n_be/i18n_extra/l10n_be.pot

```diff
@@ -1,1664 +1,1847 @@
 # Translation of Odoo Server.
 # This file contains the translation of the following modules:
-#	* l10n_be
+# 	* l10n_be
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 12.0\n"
+"Project-Id-Version: Odoo Server 13.0alpha1+e\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2019-01-17 08:59+0000\n"
-"PO-Revision-Date: 2019-01-17 08:59+0000\n"
-"Last-Translator: <>\n"
+"POT-Creation-Date: 2019-08-01 07:15+0000\n"
+"PO-Revision-Date: 2019-08-01 07:15+0000\n"
+"Last-Translator: \n"
 "Language-Team: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: \n"
 "Plural-Forms: \n"
 
 #. module: l10n_be
+#: model:account.tax,name:l10n_be.1_attn_VAT-IN-V83-00
```

Comment:
don't add this, this is the point of the l10n_multilang module, you translate the templates, not the generated records